scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -1022,6 +1022,40 @@ boolean detachService(JobId jobId, int serviceId) throws UnknownJobException {
         }
     }
 
+    boolean addExternalEndpointUrl(JobId jobId, String externalEndpointUrl) throws UnknownJobException {
+        JobData jobData = lockJob(jobId);
+        if (jobData == null) {
+            throw new UnknownJobException(jobId);
```

Comment:
as you can see, if the job is terminated (in that case, it is nor present in LiveJobs), an UnknownJobException will be thrown.

Reply:
Nothing to do but thanks a lot for the story ;)